index.html

```diff
@@ -33,10 +33,12 @@
           <li><a href="#tabWeaponsShields" data-toggle="tab">Weapons/Shields</a></li>
           <li><a href="#tabArmors" data-toggle="tab">Armor</a></li>
           <li><a href="#tabInformation" data-toggle="tab">Information</a></li>
-          <li><a href="#tabHelp" data-toggle="tab">Help & Options</a></li>
+		  <li><a href="#tabHelp" data-toggle="tab"><span class="glyphicon glyphicon-question-sign"></span></a></li>
+		  <li><a href="#tabOptions" data-toggle="tab"><span class="glyphicon glyphicon-cog"></span></a></li>
+		  </div>
         </ul><!-- /.nav.navbar-nav -->
 
-      </div><!-- /.navbar-collapse -->
+      <!--</div><!-- /.navbar-collapse -->
```

Comment:
Why was this div commented? Additionally, you are now embedding a comment in a comment